### DISCUSSION TITLE
Autonyms in Botanical publications

### DISCUSSION THREAD

**Discussion by flsimoes** [plazi/community#99] [General]
Autonyms, by definition, do not have authorities.
Is there any downstream effect (such as when transiting to GBIF) of lacking such information?

Here's an example:

78567A4BFF8EBB4AFFADFFC0FFC51B51
![image](https://user-images.githubusercontent.com/3343086/141465003-1b74f05a-e08d-4e94-afe8-0024acfe691c.png)

**Comment by gsautter** [comment]
Well, QC will complain about the lack of authority information, but if you false positive that error (actually rather declaring it a "Data Error" or data peculiarity in this instance, but to the same avail), the gatekeeper will no longer complain about this, and it won't get in the way of any transits.
The one thing responsible for any gatekeeper intervention is the error protocol, apart from bibliographic metadata checks autonomously performed by several components (mostly introduced pre-QC and pre-gatekeeper).
